resources/lang/fr/disposable.php

```diff
@@ -3,78 +3,75 @@
-    'notes'       => 'Remarques',
     'orig'        => 'Orig',
     'origin'      => 'Origine',
     'registered'  => 'Enregistré',
-    'show_hide'   => 'Voir / Cacher',
+    'show_hide'   => 'Carte / Liste',
     'total'       => 'Total',
     'weather'     => 'Météo',
-    'pagination'  => 'afficher :first premier au :last du total :total des enregistrements',
+    'pagination'  => 'Rapports :first à :last sur :total soumis',
     // Flights
-    'bid_add'     => 'Ajouter une Enchère',
-    'bid_rem'     => 'Retirer une Enchère',
+    'bid_add'     => 'Ajouter aux projets',
+    'bid_rem'     => 'Retirer des projets',
     'load_acars'  => 'Charger dans vmsAcars',
-    'new_pirep'   => 'Nouveau PIREP Manuel',
+    'new_pirep'   => 'Générer PIREP Manuel',
     'sb_generate' => 'Générer OFP SimBrief',
     'sb_view'     => 'Voir OFP SimBrief',
     // Airport Details
     '100ll_cost'     => 'Coût 100LL',
-    'avg_taxi_times' => 'Moyenne du Temps au Taxi',
-    'gh_cost'        => 'Coût de la Manutention au Sol',
+    'avg_taxi_times' => 'Moyenne de durée Taxi',
+    'gh_cost'        => 'Coût du service au Sol',
     'icao_iata_code' => 'ICAO / IATA Code',
     'jeta1_cost'     => 'Coût JET-A1',
     'mogas_cost'     => 'Coût Mogas',
     // Roster
     'awards'      => 'Récompenses',
-    'only_active' => 'Seuls les membres actifs sont répertoriés. La limite inactif est de :days jours',
+    'only_active' => 'Seuls les membres actifs sont répertoriés. Vous serez radié au bout de :days jours d\'absence.',
     'rank'        => 'Grade',
     // Pireps
-    'fused'  => 'Carburant Utilisé',
-    'lpirep' => 'Dernier détails PIREP',
-    'lrate'  => 'Landing Rate',
+    'fused'  => 'Consommé',
+    'lpirep' => 'Dernier PIREP soumis:',
+    'lrate'  => 'Tx de descente',
     'score'  => 'Score',
     // Dashboard
-    'avg_ftime'     => 'Moy.Temps de Vol',
-    'avg_fused'     => 'Moy.Carburant Utilisé',
-    'avg_lrate'     => 'Moy.Landing Rate',
-    'avg_score'     => 'Moy.Score',
-    'user_on_leave' => 'Vous êtes en congé ! Déposez un PIREP pour que votre statut devienne actif.!',
+    'avg_ftime'     => 'Durée moy. de vol',
+    'avg_fused'     => 'Consommation moyenne',
+    'avg_lrate'     => 'Descente: tx moy.',
+    'avg_score'     => 'Score Moyen',
+    'user_on_leave' => 'Vous êtes de repos ! Déposez un PIREP pour vous déclarer actif!',
     // Menu Items
-    'menu_roster'   => 'Liste des Pilotes',
+    'menu_roster'   => 'Nos Pilotes',
     'menu_airlines' => 'Compagnies',
     'menu_ranks'    => 'Grades',
     'menu_awards'   => 'Récompenses',
     'menu_stats'    => 'Statistiques',
-    'menu_fltops'   => 'Operations de vols',
+    'menu_fltops'   => 'Opérations de vols',
     'menu_flights'  => 'Vols',
-    'menu_tours'    => 'Tours',
-    'menu_fleet'    => 'Flotte',
-    'menu_hubs'     => 'Bases',
-    'menu_reports'  => 'Rapports Pilotes',
+    'menu_tours'    => 'Circuits',
+    'menu_fleet'    => 'Flottes',
```

Comment:
Are we talking about a fleet ? If it's only a fleet (singular) then there is no "s" at the end